### DISCUSSION TITLE
Is it possible to "sync" emails between separate imap accounts?

### DISCUSSION THREAD

**Discussion by shalak** [umanamente/n8n-nodes-imap#6] [Q&A] [answered]
In the README, I don't see any mention of creating/uploading an email to the mailbox. Is it supported?

My use-case is that I have 2 separate IMAP accounts and I want to sync "Sent" folders between them.

**Comment by umanamente** [comment]
Creating emails is not supported currently, but I'm considering adding this feature

**Comment by umanamente** [comment, marked as answer]
> In the README, I don't see any mention of creating/uploading an email to the mailbox. Is it supported?
> 
> My use-case is that I have 2 separate IMAP accounts and I want to sync "Sent" folders between them.


"create draft" operation added in v2.1.0

**Reply by shalak** [reply]
I understand, that "draft" emails are just regular email from the email client perspective, as long as I create them in e.g. "Sent" folder?

**Reply by umanamente** [reply]
It may depend on the IMAP server implementation. Please test it on your server and let me know if it functions as expected.
